src/ontology/mondo-edit.obo

```diff
@@ -298193,10 +298192,10 @@ xref: GARD:15783 {source="OMIM:614250"}
 xref: OMIM:614250 {source="MONDO:equivalentTo"}
 xref: Orphanet:2073 {source="OMIM:614250"}
 xref: UMLS:C3280266 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:614250"}
-is_a: MONDO:0016158 {source="Orphanet:2073/btnt"} ! narcolepsy-cataplexy syndrome
-is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:614250"} ! narcolepsy
-intersection_of: MONDO:0021107 ! narcolepsy
+is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:614250"} ! hereditary narcolepsy
+intersection_of: MONDO:0021107 ! hereditary narcolepsy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7197 ! MOG
+relationship: excluded_subClassOf MONDO:0016158 {source="Orphanet:2073/btnt", source="https://orcid.org/0000-0001-5208-3432"} ! narcolepsy-cataplexy syndrome
```

Comment:
I think this term can have 2 parents: hereditary narcolepsy AND narcolepsy-cataplexy